examples/pbam/toy_dynamics.in

```diff
@@ -3,24 +3,26 @@ read
     mol pqr neg_charge.pqr
 end
 elec name comp_solv        # Toy charges
-    pbam
+    pbam-auto
 
     runtype dynamics                                   # Can be energyforce, electrostatics etc
     runname dyn_toy                                    # Output name for files created by program
     temp 298.15                                        # System temperature (K)
     pdie 4.0                                           # Solute dielectric                 
     sdie 78.0                                          # Solvent dielectric                
-    ion charge 0.0 conc 0.05 radius 0.0                # Monovalent salt conc in M
+    # WARN: ion is no longer supported in pbam-auto
+    # ion charge 0.0 conc 0.05 radius 0.0                # Monovalent salt conc in M
 
     pbc 100.0
 
-    termcombine or
-    term time 550.0
-    term position r>= 81.8 1
+    # termcombine or
+    # term time 550.0
+    # term r>= 81.8 1
     ntraj 3
 
-    diff 1 move 0.01 0.001
-    diff 2 stat
+    # TODO: FIX:
+    # diff 1 move 0.01 0.001
+    # diff 2 stat
```

Comment:
I forgot to uncomment this example after fixing the grammar for diff. It is fixed now.